### PR TITLE
Interface > interface groups

### PR DESCRIPTION
**Pull request by thanegill**
TODO:
- Hide id label on re-edit #10 
- Add cancel button to edit #11

### CODE REVIEW

File: usr/local/www/interfaces_groups_edit.php [hunk 1]
```diff
@@ -36,7 +36,7 @@
 ##|+PRIV
 ##|*IDENT=page-interfaces-groups-edit
 ##|*NAME=Interfaces: Groups: Edit page
-##|*DESCR=Allow access to the 'Interfaces: Groups: Edit' page.
+##|*descr=Allow access to the 'Interfaces: Groups: Edit' page.
```
Comment (SjonHortensius) [inline usr/local/www/interfaces_groups_edit.php]: Pretty sure this is unintended

File: usr/local/www/interfaces_groups_edit.php [hunk 2]
```diff
@@ -168,193 +172,76 @@
...
+display_top_tabs($tab_array);
+
+require('classes/Form.class.php');
+$form = new Form;
+$section = new Form_Section('Interface Group Edit');
+
+$section->addInput(new Form_Input(
+	'Group Name',
+	'text',
+	htmlspecialchars($pconfig['ifname']),
```
Comment (SjonHortensius) [inline usr/local/www/interfaces_groups_edit.php]: No need for manual htmlspecialchars calls, all Inputs will handle this for you

File: usr/local/www/interfaces_groups_edit.php [hunk 3]
```diff
@@ -168,193 +172,76 @@
...
+	['placeholder' => 'Group Name']
+))->setHelp(gettext('No numbers or spaces are allowed. Only '.
```
Comment (SjonHortensius) [inline usr/local/www/interfaces_groups_edit.php]: No need for manual gettext(), those are also handled by the Inputs

File: usr/local/www/interfaces_groups_edit.php [hunk 4]
```diff
@@ -168,193 +172,76 @@
...
+	['placeholder' => 'Group Description']
+))->setWidth(6)->setHelp(gettext('You may enter a group decsription here '.
+'for your reference (not parsed)'))->forceName('descr');
+
+$section->addInput(new Form_Select(
+	'Group Members',
+	explode(' ', $pconfig['members']),
+	$interface_list,
+	true
+))->setWidth(6)->setHelp('<strong>' . gettext('NOTE: ') . '</strong>'.
```
Comment (SjonHortensius) [inline usr/local/www/interfaces_groups_edit.php]: Please remove the `<strong>`; the css should determine styling

File: usr/local/www/interfaces_groups_edit.php [hunk 5]
```diff
@@ -168,193 +172,76 @@
...
-	rows = 1;
-	totalrows = <?php echo $counter; ?>;
-	loaded = <?php echo $counter; ?>;
-//]]>
-</script>
+$form->setSubmit(new Form_Input(
+	'Submit',
+	'submit',
+	gettext('Save')
+))->forceName('Submit')->removeClass('form-control')->addClass('btn btn-primary');
```
Comment (SjonHortensius) [inline usr/local/www/interfaces_groups_edit.php]: This is not needed, Form::__construct takes care of this itself. If you remove this block nothing will change.

**Comment by thanegill** [comment]
Fixed all needed changes

**Pull request merged**